src/main/resources/banner.txt

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
no.

Reply:
Action based PR comments 🙏🏽

Reply:
we should run without a startup banner at all, as each line can end up in application insights.